src/MaaWpfGui/ViewModels/UI/AnnouncementViewModel.cs

```diff
@@ -26,6 +32,91 @@ namespace MaaWpfGui.ViewModels.UI
     // ReSharper disable once ClassNeverInstantiated.Global
     public class AnnouncementViewModel : Screen
     {
+        public string ImageSource { get; set; }
+
+        public class AnnouncementSection
+        {
+            public string Title { get; set; }
+
+            public string Content { get; set; }
+        }
+
+        public AnnouncementViewModel()
+        {
+            UpdateImageSource();
+
+            UpdateScrollStateCommand = new RelayCommand<ScrollViewer>(scrollViewer =>
+            {
+                if (scrollViewer == null)
+                {
+                    return;
+                }
+
+                // 计算是否滚动到底部
+                IsScrolledToBottom |= scrollViewer.VerticalOffset >= scrollViewer.ScrollableHeight - 10;
+            });
+            AnnouncementSections = new(
+                ParseAnnouncementInfo(AnnouncementInfo));
+            SelectedAnnouncementSection = AnnouncementSections.FirstOrDefault();
+        }
+
+        private void UpdateImageSource()
+        {
+            ImageSource = SettingsViewModel.GuiSettings.Language switch
+            {
+                "zh-cn" or "zh-tw" => "/Res/Img/NoSkland.jpg",
+                _ => "/Res/Img/NoSkLandEn.jpg",
+            };
+        }
+
+        private static ObservableCollection<AnnouncementSection> ParseAnnouncementInfo(string markdown)
+        {
+            var sections = markdown.Split(["### "], StringSplitOptions.RemoveEmptyEntries)
+                .Select(section =>
+                {
+                    var lines = section.Split('\n');
+                    return new AnnouncementSection
+                    {
+                        Title = lines.FirstOrDefault(),
+                        Content = string.Join("\n", lines.Skip(1)).TrimEnd('-'),
+                    };
+                }).ToList();
+
+            sections.Insert(0, new()
+            {
+                Title = "ALL~ the Announcements",
```

Comment:
```suggestion
                Title = "All the Announcements",
```
Even though the ideal solution would be to have this **bold**